lib/jerboa/client.ex

```diff
@@ -1,28 +1,76 @@
 defmodule Jerboa.Client do
   @moduledoc """
-  STUN client
+  STUN client process
+
+  `start/1` function is used to spawn new client process:
+
+      iex> Jerboa.Client.start server: %{address: server_ip, port: server_port}
+      {:ok, #PID<...>}
+
+  (see `start/1` for configuration options)
+
+  Currently the only implemented STUN method is Binding, which allows client to
```

Comment:
Unfortunately I can't find a part explicitly describing the binding method in the RFC.